### PR TITLE
Part 1 of Pepper-587 broken down to small PRs

### PR DESCRIPTION
**Pull request by Rinchan20**
can read .env file now - this PR is part 1 of this PR (https://github.com/broadinstitute/ddp-angular/pull/2048) broken down to nice, smaller ones

### CODE REVIEW

**Comment by Rinchan20** [comment]
adds the tests that checks that a newly created RGP study participant that completes the enrollment survey can be found in DSM

**Comment by Rinchan20** [comment]
note: at least half of the new files are from the screenshot comparison test(s) in RGP child enrollment test (ran to make sure additions don't break previous tests)

File: playwright-e2e/dsm/pages/participant-list-page.ts [hunk 1]
```diff
@@ -136,6 +136,31 @@ export default class ParticipantListPage {
...
+  await searchPanel.text('Participant ID', {textValue: participantGUID });
+  await searchPanel.search();
+}
+
+/**
+ * Selects a specific participant from the participant list when given their guid
+ * @param participantGUID the guid of the specific participant to search for
+ */
+public async selectParticipant(participantGUID: string): Promise<void> {
+  await this.page.getByRole('cell', { name: participantGUID }).click()
```
Comment (aweng98) [inline playwright-e2e/dsm/pages/participant-list-page.ts]: Use [openParticipantPageAt(position: number)](https://github.com/broadinstitute/ddp-angular/blob/03a9f2eed954ecd8f99cdf924be8fb6c2807977b/playwright-e2e/dsm/component/tables/participant-list-table.ts#L31)
I don't think you need to write this func.

**Review by aweng98 — APPROVED** [review]
LGTM. Thank you for making this simple to review.

**Pull request merged**